plugins/pixart-rf/pixart-rf.quirk

```diff
@@ -63,6 +63,36 @@ GType = FuPxiBleDevice
 Plugin = pixart_rf
 GType = FuPxiBleDevice
 
+# Elecom Bluetooth Keyboard
+[HIDRAW\VEN_056E&DEV_1095]
+Plugin = pixart_rf
+GType = FuPxiBleDevice
+
+# Targus BT Mouse AMB844
+[HIDRAW\VEN_1048&DEV_1013]
+Plugin = pixart_rf
+GType = FuPxiBleDevice
+
+# AKR123 Bluetooth Keyboard
+[HIDRAW\VEN_04F2&DEV_2199]
+Plugin = pixart_rf
+GType = FuPxiBleDevice
+
+# AMR120 Bluetooth Mouse
+[HIDRAW\VEN_04F2&DEV_2198]
+Plugin = pixart_rf
+GType = FuPxiBleDevice
+
+# j5 Wireless Keybaord
```

Comment:
You mean `Keyboard` here

Reply:
Sorry, this is typo. Thanks for your checked.
Do I need re-commit this error?